doc/handbook.md

```diff
@@ -99,12 +102,135 @@ or patch versions. As a rule of thumb:
 directory, using the name of the executor as file extension. For example, the
 Dockerfile for the `deploy` executor of the `helm` orb is available at
 `orbs/helm/Dockerfile.deploy`.
+
+### Orbs creation
+
+#### **Prerequisites:**
+
+- Clone repo [https://github.com/jobteaser/circleci](https://github.com/jobteaser/circleci)
+- Install circleci cli: [https://circleci.com/docs/2.0/local-cli/](https://circleci.com/docs/2.0/local-cli/)
+
+List of orbs available in JT: [https://circleci.com/developer/orbs?query=jobteaser](https://circleci.com/developer/orbs?query=jobteaser)
+
+File used to illustrate in this doc: [https://github.com/jobteaser/circleci/blob/master/orbs/e2e-web/orb.yml](https://github.com/jobteaser/circleci/blob/master/orbs/e2e-web/orb.yml)
+
+#### **Guide**
+
+- On a branch, create a new folder on orbs repository with the name of your custom orb
+
+- Create a new file _orb.yml_ in your folder. This file will be your orb configuration file.
+
+**Example**: to create a new orb “e2e-web”, create folder “e2e-web” with your orb.yml file
+
+- In this _orb.yml_ file, define your needed commands, jobs, executor… You can also use parameters which will be used when you will call your orb.
+
+**Example:**
+
+```jobs:
+  execute_e2e_tests:
+    parameters:
+      tags:
+        type: string
+        default: "not @flaky and not @wip and not @browserstack and not @prod"
+      max_instances:
+        type: string
+        default: "30"
+      command:
+        description: "The command to run in e2e_jt_tests repo"
+        type: "string"
+        default: "npm run chrome"
+      e2e_cookie:
+        description: "The cookie to add to bypass rate limit"
+        type: "string"
+        default: $COOKIE_E2E
+    executor:
+      name: "default"
+      tags: << parameters.tags >>
+      max_instances: << parameters.max_instances >>
+    steps:
+      - service/configure_ssh
+      - clone_test_suite
+      - manage_dependencies
+      - create_report_folder
+      - run_e2e_test:
+          command: << parameters.command >>
+          e2e_cookie: << parameters.e2e_cookie >>
+      - generate_reporting
+      - slack_alerting
+```
+
+In this case, I have created a job, with parameters. Steps I used are also custom commands I have in my [_orb.yml_](https://github.com/jobteaser/circleci/blob/master/orbs/e2e-web/orb.yml) file
+
+When your file is configured, you have to create your orb in the repo. Open your terminal on your circleci repo and type:
+
+```circleci orb create jobteaser/orbName
+```
+
+_orbName_ has to be replaced with your orb name (ie your folder name)
+
+**Example:** circleci orb create jobteaser/e2e-web
+
+If you have: Error: AUTHORIZATION_FAILURE ask **devex** team to do it for you
+
+- To test your orb, you can publish it with a custom name to be used in your other repo which will call the orb.
+
+**Example:**
+
+```circleci orb publish orbs/<<orbName>>/orb.yml jobteaser/<<orbName>>@dev:orb_e2e_tests
```

Comment:
I think you may need to add a linefeed here after the triple "`" (same below), because the preview of your md does not show this content.